### PR TITLE
W-16560355 ACM doc updates for analytics EOL jh

### PR DESCRIPTION
**Pull request by JennyHajee**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 1]
```diff
@@ -27,6 +27,16 @@ If you create a role, enter a role name such as `Salesforce_ACM_org_admin_user`.
...
 . Click *Save*.
 
+== Add the Monitoring Viewer Permission
+Add the Monitoring Viewer permission to view metrics for applications in the *My Applications* details page:
+
+. Log in to Anypoint Platform and go to *Access Management*.
+. Select *Teams* and select *Community Users*.
+. Select the *Permissions* tab and click *Add permissions*.
+. Select *Anypoint Monitoring* > *Monitoring Viewer* and click *Next*.
+. Select all business groups and click *Next*.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/installation-prerequisites.adoc]: All business groups that applies

File: modules/ROOT/pages/troubleshooting-metrics-not-showing-for-applications.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+= Metrics Not Showing for Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+If the metrics aren't showing for client applications on the *My Applications* details page, ensure that you have the *Monitoring Viewer* permissions.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/troubleshooting-metrics-not-showing-for-applications.adoc]: One reason could be this one. they could have another issue

**Review by ferferrarazzo — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/troubleshooting-metrics-not-showing-for-applications.adoc (hunk 2 above)
Comment (JennyHajee) [reply modules/ROOT/pages/troubleshooting-metrics-not-showing-for-applications.adoc]: Hi @ferferrarazzo Should we document other reasons?

**Review by ferferrarazzo — DISMISSED** [review]

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 3]
```diff
@@ -27,6 +27,16 @@ If you create a role, enter a role name such as `Salesforce_ACM_org_admin_user`.
 . Enable the option *Salesforce CRM Content User*.
 . Click *Save*.
 
+== Add the Monitoring Viewer Permission
+Add the Monitoring Viewer permission to view metrics for applications in the *My Applications* details page:
```
Comment (tasmoore) [inline modules/ROOT/pages/installation-prerequisites.adoc]:
```suggestion
Add the Monitoring Viewer permission to view metrics for applications on the *My Applications* details page:
```

**Review by tasmoore — DISMISSED** [review]

**Review by tasmoore — APPROVED** [review]

**Pull request merged**